Tests/TestFramework/TestRunnerTest.cpp, Line 365
Test Main has empty path

Expected Exception Type:
[std::runtime_error]
output/TestsTemporaryData/TestFramework/TestRunnerTest/Auxiliary/FakeProject/  is not a regular file

Try ensuring that the application is run from the appropriate directory

=======================================

Tests/TestFramework/TestRunnerTest.cpp, Line 374
Test Main does not exist

Expected Exception Type:
[std::runtime_error]
output/TestsTemporaryData/TestFramework/TestRunnerTest/Auxiliary/FakeProject/FooMain.cpp not found

Try ensuring that the application is run from the appropriate directory

=======================================

Tests/TestFramework/TestRunnerTest.cpp, Line 383
Include Target has empty path

Expected Exception Type:
[std::runtime_error]
output/TestsTemporaryData/TestFramework/TestRunnerTest/Auxiliary/FakeProject/  is not a regular file

Common includes not found

=======================================

Tests/TestFramework/TestRunnerTest.cpp, Line 392
Include Target does not exist

Expected Exception Type:
[std::runtime_error]
output/TestsTemporaryData/TestFramework/TestRunnerTest/Auxiliary/FakeProject/FooPath.hpp not found

Common includes not found

=======================================

Tests/TestFramework/TestRunnerTest.cpp, Line 401
Project root is empty

Expected Exception Type:
[std::runtime_error]
Unable to locate project root as no commandline arguments supplied.

=======================================

Tests/TestFramework/TestRunnerTest.cpp, Line 409
Project root does not exist

Expected Exception Type:
[std::runtime_error]
canonical: unable to find path

=======================================

Tests/TestFramework/TestRunnerTest.cpp, Line 418
Project root not findable

Expected Exception Type:
[std::runtime_error]
Unable to locate project root from path:
output/TestsTemporaryData/TestFramework/TestRunnerTest/Auxiliary/FakeProject/TestShared
Please ensure that the build directory is a subdirectory of <project>/build.

=======================================

Tests/TestFramework/TestRunnerTest.cpp, Line 428
Neither name nor source unique

Expected Exception Type:
[std::runtime_error]
  Error: Suite/Test: "Duplicates/Free Test"
Source file: ".../Tests/TestFramework/TestRunnerTest.cpp"
Please do not include tests in the same suite which both have the same name and are defined in the same source file.

=======================================

Tests/TestFramework/TestRunnerTest.cpp, Line 462
Invalid repetitions for instability analysis

Expected Exception Type:
[std::runtime_error]
locate-instabilities: unable to interpret 'foo' as an integer number of repetitions

=======================================

Tests/TestFramework/TestRunnerTest.cpp, Line 469
Insufficient repetitions for instability analysis

Expected Exception Type:
[std::runtime_error]
  Error: Number of repetitions, must be >= 2

=======================================

